src/ManagedDripsHub.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.7;
+
+import {UUPSUpgradeable} from "openzeppelin-contracts/proxy/utils/UUPSUpgradeable.sol";
+import {ERC1967Proxy} from "openzeppelin-contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import {ERC1967Upgrade} from "openzeppelin-contracts/proxy/ERC1967/ERC1967Upgrade.sol";
+import {StorageSlot} from "openzeppelin-contracts/utils/StorageSlot.sol";
+import {DripsHub, SplitsReceiver} from "./DripsHub.sol";
+import {ERC1967Pausable} from "./ERC1967Pausable.sol";
+
+/// @notice The DripsHub which is UUPS-upgradable, pausable and has an owner.
+/// It can't be used directly, only via a proxy.
+///
+/// ManagedDripsHub uses the ERC-1967 admin slot to store the owner address.
+/// While this contract is capable of updating the owner,
+/// the proxy is expected to set up the initial value of the ERC-1967 admin.
+abstract contract ManagedDripsHub is DripsHub, UUPSUpgradeable, ERC1967Pausable {
```

Comment:
I think the `MangedDripsHub` is a bit over engineered. We could remove it and just make
```solidity
ERC20DripHub is DripsHub, UUPSUpgradeable, ERC1967Pausable

```

What do you think?

Reply:
We'd need to do that with `EthDripsHub` too. IMO it's the right way to organize code, because it encapsulates some important bits, which don't really match the `ERC20` part of `ERC20DripsHub` and could get obscured in the token logic. After all the goal isn't to have as few files as possible.

Reply:
Not relevant anymore with new design. Resolved.